src/Native/Windows/build-native.cmd

```diff
@@ -29,6 +30,9 @@ if /i [%1] == [/p:Platform]    (
     echo Error: Invalid platform args "%1 and %2"
     exit /b 1
 )
+if /i [%1] == [/p:OfficialBuildId] (
```

Comment:
Why do you need to parse this in the cmd file directly as opposed to just passing it through to msbuild?